install_dependencies.sh

```diff
@@ -32,7 +32,29 @@ echo "Installing Misc. Utilities"
 echo "================================================================"
 
 sudo apt-get install -y\
-    clang-format
+    clang-format\
+    python-rosinstall
+
+
+echo "================================================================"
+echo "Installing Project Dependent ROS packages."
+echo "================================================================"
+
+# Setup rosinstall
+sudo mkdir -p /usr/share/ros/
+sudo chmod a+rwx /usr/share/ros
+rosinstall .
+echo "bash $CURR_DIR/setup.sh" >> ~/.bashrc
+
+echo "================================================================"
+echo "Installing Udev rules for phidgets"
+echo "================================================================"
+
+# Setup udev rules
+sudo cp /usr/share/ros/phidgets_api/share/udev/99-phidgets.rules /etc/udev/rules.d
+echo "Phidgets udev rules have been copied to /etc/udev/rules.d"
```

Comment:
I left a similar comment over on `Jormungandr`, but could we submit an issue/pr to the `phidget` repo, and make an issue to remove this section from `install_dependencies.sh` once that's been fixed/merged?

Reply:
This is generally not a problem I think because it's run manually. Prob as easy as changing the bash v. source.